### PR TITLE
Issue-560

### LINKED ISSUE #560: FAQ: How to get each agent message?
https://microsoft.github.io/autogen/docs/reference/agentchat/conversable_agent#chat_messages

_Originally posted by @sonichi in https://github.com/microsoft/autogen/issues/478#issuecomment-1784210201_
            
The answer should be added to FAQ, together with the question "how to get the last message?"

### PR DESCRIPTION
**Pull request by olaoluwasalami**
@sonichi @thinkall 

## Why are these changes needed?

Add to FAQ - How to get each agent message?

## Related issue number

Closes #560 

## Checks

- [x] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [x] I've made sure all auto checks have passed.

### CODE REVIEW

File: website/docs/FAQ.md [hunk 1]
```diff
@@ -127,3 +127,7 @@ Workaround:
 2. `mkdir /home/vscode/.local/lib/python3.10/site-packages/google/colab`
 
 Explanation: Per [this gist](https://gist.github.com/defulmere/8b9695e415a44271061cc8e272f3c300?permalink_comment_id=4711478#gistcomment-4711478), linked from the official [chromadb docs](https://docs.trychroma.com/troubleshooting#sqlite), adding this folder triggers chromadb to use pysqlite3 instead of the default.
+
+## How to get last message ?
+
+Please refer to https://microsoft.github.io/autogen/docs/reference/agentchat/conversable_agent#chat_messages
```
Comment (sonichi) [inline website/docs/FAQ.md:133]: The question and answer don't match. There are two methods, one for getting last message and the other for getting all the messages.

Comment (olaoluwasalami) [reply website/docs/FAQ.md]: Thanks, changed made to last message

**Review by sonichi — COMMENTED** [review]
#560 is about two questions. This PR only addresses one.

**Comment by olaoluwasalami** [comment]
@sonichi  PR updated

**Review by gagb — APPROVED** [review]
PR adds two FAQ questions and answers that close an existing issue.

**Pull request merged**